scripts/update-user-origin.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+require 'json'
+
+def usage
+  <<~USAGE
+    Usage: #{$0} desired-origin user-guid
```

Comment:
_Nitpick_ - wouldn't it be more semantically correct if it were `#{$0} user-guid desired-origin` - the aim of the script is to set the origin of the user, not modify the origin to include that user?

Reply:
True, I should use flags :trollface: 

Updated